packages/expo-tracking-transparency/android/src/main/java/expo/modules/trackingtransparency/TrackingTransparencyModule.kt

```diff
@@ -0,0 +1,25 @@
+package expo.modules.trackingtransparency
+
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+import com.google.android.gms.ads.identifier.AdvertisingIdClient
+import android.content.Context
+import expo.modules.kotlin.exception.Exceptions
+
+class TrackingTransparencyModule : Module() {
+  private val context: Context
+    get() = appContext.reactContext ?: throw Exceptions.ReactContextLost()
+
+  override fun definition() = ModuleDefinition {
+    // TODO: Rename the package to 'ExpoTracking'
+    Name("ExpoTrackingTransparency")
+
+    Function("getAdvertisingId") {
+      return@Function getAdvertisingId()
+    }
+  }
+
+  private fun getAdvertisingId(): String? {
+    return AdvertisingIdClient.getAdvertisingIdInfo(context).id
```

Comment:
@tsapeta can you please confirm this will get converted to null in JS (not undefined)?